lib/suma/twilio.rb

```diff
@@ -41,8 +41,12 @@ def self.update_verification(ve_id, kw)
       update(**kw)
     return response
   rescue Twilio::REST::RestError => e
-    # ignore 404s, it means twilio has approved, expired or invalidated the code already
+    # ignores 20404s, it means twilio has approved, expired or invalidated the code already
     # https://www.twilio.com/docs/verify/api/verification-check#check-a-verification
-    raise(e) unless e.code === 404
```

Comment:
As a note- Twilio automatically set the `code` to `404`(http status) if there's no `code` in the body. That is why this test worked previously, even though `body.code` wasn't passed by the test.